citizenvoice/survey_design/models/category.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.utils.text import slugify
+from django.utils.translation import gettext_lazy as _
+
+from .survey import Survey
+
+
+class Category(models.Model):
+
```

Comment:
Could you add a description of what the Category is as docstring? Even if it is not our code, it is a good practice we comment on the code so that other developers will understand it